capabilityquerier/cmd/main.go

```diff
@@ -87,19 +71,23 @@ func queryEndpoints(message []byte, args *map[string]interface{}) error {
 
 	metadataURL.Path = path.Join(metadataURL.Path, "metadata")
 
-	job := capabilityquerier.Job{
-		Context:      ctx,
-		Duration:     jobDuration,
-		FHIRURL:      metadataURL,
-		Client:       client,
-		MessageQueue: mq,
-		Channel:      ch,
-		QueueName:    qName,
+	jobArgs := make(map[string]interface{})
+	jobArgs["FHIRURL"] = metadataURL
+	jobArgs["client"] = (*args)["client"]
+	jobArgs["mq"] = (*args)["mq"]
+	jobArgs["ch"] = (*args)["ch"]
+	jobArgs["qName"] = (*args)["qName"]
```

Comment:
thoughts on creating an object rather than passing in each item as its own thing?
probably not worth it... but may make it cleaner/less error prone.